src/MigrationTools/ProcessorEnrichers/PauseAfterEachItem.cs

```diff
@@ -14,8 +15,11 @@ public PauseAfterEachItemOptions Options
             get { return _Options; }
         }
 
-        public PauseAfterEachItem(IMigrationEngine engine, ILogger<WorkItemProcessorEnricher> logger) : base(engine, logger)
+        public object Engine { get; private set; }
```

Comment:
why object, and what are you using it for?

Reply:
No, it was generated... needs changed to IMigrationEngine